en/installation/deploying-on-heroku.md

```diff
@@ -5,13 +5,13 @@
 This tutorial assumes that you have already managed to clone CONSUL on your machine and gotten it to work.
 
 1. First, create a [Heroku](https://www.heroku.com) account if it isn't already done.
-2. Install the [Heroku CLI](https://devcenter.heroku.com/articles/heroku-cli) and sign in using
+1. Install the [Heroku CLI](https://devcenter.heroku.com/articles/heroku-cli) and sign in using
 
   ```bash
   heroku login
   ```
 
-3. Go to your CONSUL repository and instantiate the process
+1. Go to your CONSUL repository and instantiate the process
```

Comment:
Applying the "Ordered list item prefix" in places like this one breaks the generated HTML since it doesn't generate one list but several ones due to the fact that we include paragrahps and code between one lement of the list and the next one :relieved:.

If we really want to apply this rule, we need to make it so we only generate one list and we include paragraphs and code inside `<li>` elements.

Reply:
Ok, I'll check it out. Although in my text editor the "preview" is displayed correctly.

Reply:
In the end I decided to keep the original version and comment on it in the commit message.

Reply:
After thinking about it a bit, I think we should either fix the places having this issue so we only generate one `<ol>` tag or remove the "Ordered list item prefix" rule (and all its related changes in this pull request). It's true that we've got this rule in Consul, but it isn't applied anywhere since we don't define ordered lists there :sweat_smile:.

With the current changes regarding this rule, we're making the code more inconsistent, which is the opposite of what linters are for.

Reply:
In the end I removed the commit that made the changes for the "MD029 Ordered list item prefix" rule and added another commit to remove the rule.